syllabus/4-Smart_Contracts/6-Wasm_ink!_slides.md

```diff
@@ -434,6 +423,12 @@ Notes:
 
 ---
 
+<!-- ## typechain-compiler -->
+
+<!-- <\!-- TODO: typechain-compiler -\-> -->
+
+<!-- --- -->
```

Comment:
same as above
It would be preferable not to have left-overs TODO

Reply:
removed